palace/linalg/errorestimator.cpp

```diff
@@ -23,14 +22,13 @@ namespace palace
 namespace
 {
 
-std::unique_ptr<Operator>
-GetMassMatrix(const mfem::ParFiniteElementSpaceHierarchy &fespaces, int pa_order_threshold,
-              int skip_zeros)
+std::unique_ptr<Operator> GetMassMatrix(const FiniteElementSpaceHierarchy &fespaces,
+                                        int pa_order_threshold, int skip_zeros)
```

Comment:
Nit: `int skip_zeros` can be bool no?

Reply:
Whoops, #120 fixed these but rebasing messed things up. Fixed in [8c05eaf](https://github.com/awslabs/palace/pull/118/commits/8c05eaf6638ac19cd287a7c347987c3f96a8fb54).